catalog/ui/catalog-ui-search/pom.xml

```diff
@@ -341,7 +346,6 @@
                             !org.abego.treelayout.*,
                             *
                         </Import-Package>
-                        <Export-Package/>
```

Comment:
This should not be removed, as by default the Export-Package will export ALL things (minus a few things). Check out defaults here: http://felix.apache.org/documentation/subprojects/apache-felix-maven-bundle-plugin-bnd.html#default-behavior